src/main/webapp/src/app/user-profile/user-profile.component.ts

```diff
@@ -81,11 +81,10 @@ export class UserProfileComponent implements OnInit {
         // Successful responses are handled here.
         this.displayInfo = response.body;
       },
-      error: error => {
+      error: () => {
```

Comment:
Why obscure all errors that occur? This could introduce some obstacles when debugging.

Reply:
Will modify this handler to better handle specific errors. I will address your previous comment as a part of addressing this comment.